docroot/profiles/custom/cgov_site/modules/custom/cgov_core/config/install/core.date_format.cgov_short_date.yml

```diff
@@ -0,0 +1,7 @@
+langcode: en
```

Comment:
Can this file be reverted if it is not needed anymore? I think it is left over from Blair's PR which was missing the timestamp.